src/extension.js

```diff
@@ -142,6 +144,51 @@ function activate(context) {
 		return html;
 	}
 
+	function projectNamesHandle(config) {

```

Comment:
Could you explain why adding this?

Reply:
Well, this probably has nothing to do with the core functionality of this plugin. It looks like an Easter egg.